source/_integrations/homekit.markdown

```diff
@@ -596,6 +596,8 @@ See [resetting accessories](#resetting-accessories)
 
 Unfortunately, that sometimes happens at the moment. It might help to close the `Home` App and delete it from the cache. Usually, the accessory should get back to responding after a few minutes at most.
 
+There are reports where the IGMP settings of a router were causing issues. See [HomeKit - No Response](https://community.home-assistant.io/t/homekit-no-response-solved/170981) and [HomeKit “No Response”](https://community.home-assistant.io/t/homekit-no-response/170974) for more information.
```

Comment:
Instead of referring to external resources and discussions, we should actually document it.